climada/hazard/test/test_storm_europe.py

```diff
@@ -127,21 +127,21 @@ def test_set_ssi(self):
         storms = StormEurope.from_footprints(WS_DEMO_NC)
 
         storms.set_ssi(method='dawkins')
-        ssi_dawg = np.asarray([1.44573572e+09, 6.16173724e+08])
+        ssi_dawg = np.asarray([0.18808288, 0.08014435])
```

Comment:
@ThomasRoosli : are the changed values ok?

Reply:
The large change in magnitude seems unexpected to me. I will try to investigate somewhen soon.

Reply:
So, I think there is a mistake in the proposed implementation and the tests in StormEurope should not be changed this drastically.
Here is a small code example:
```
from climada.hazard import StormEurope
from climada.util.constants import WS_DEMO_NC

storms = StormEurope.from_footprints(WS_DEMO_NC)
storms.centroids.area_pixel
# current: array([12042728.52494331, 12042728.52494315, 12042728.52494331, ..., 11142431.39844684, 11142431.39844654, 11142431.39844684])
# newly proposed in pr484: array([0.00152588, 0.00152588, 0.00152588, ..., 0.00152588, 0.00152588, 0.00152588])
```
The centroids created with the lines of code above have a resolution of about 4 km at the equator which would mean around 16000000 m^2 area per pixel at the equator. The function `set_area_pixel` aims to return the area per pixel in m*m and to me the magnitude looks somewhat reasonable in the current implementation. With the newly proposed changes the area is below 1 m^2 per pixel which is weird.

@emanuel-schmid : please revert the changes